pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -1,14 +1,64 @@
 package v1
```

Comment:
move all helper funcs out of API pkg, or just those that need the ign config types?

Reasoning: We cannot migrate anything that depends on ignition config over to api repo